WordPress/WordPressTest/WPAnalyticsTrackerAutomatticTracksTests.m

```diff
@@ -0,0 +1,61 @@
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+#import "WPAnalyticsTrackerAutomatticTracks.h"
+#import <Automattic-Tracks-iOS/TracksService.h>
+#import <OCMock/OCMock.h>
+
+@interface WPAnalyticsTrackerAutomatticTracksTests : XCTestCase
+
+@property (nonatomic, strong) WPAnalyticsTrackerAutomatticTracks *subject;
+@property (nonatomic, strong) id serviceMock;
+
+@end
+
+@interface WPAnalyticsTrackerAutomatticTracks (Testing)
+
+@property (nonatomic, strong) TracksService *tracksService;
+@property (nonatomic, strong) TracksContextManager *contextManager;
+
+@end
+
+@implementation WPAnalyticsTrackerAutomatticTracksTests
+
+- (void)setUp {
+    [super setUp];
+
+    self.subject = [[WPAnalyticsTrackerAutomatticTracks alloc] init];
+    self.serviceMock = OCMStrictClassMock([TracksService class]);
+    self.subject.tracksService = self.serviceMock;
+    self.subject.contextManager = nil;
+}
+
+- (void)tearDown {
+    [super tearDown];
+    
+    self.subject = nil;
+    self.serviceMock = nil;
+}
+
+- (void)testVerifyTracksNamesMappings {
+    for (NSUInteger x = 0; x <= WPAnalyticsStatTwoFactorSentSMS; ++x) {
```

Comment:
No idea if i was the first one, but yeah, i usually add a sentinel there. Makes things prettier :smile: